grammar/en/src/rules_datetime.rs

```diff
@@ -198,11 +175,11 @@ pub fn rules_datetime(b: &mut RuleSetBuilder<Dimension>) -> RustlingResult<()> {
                       |_| helpers::cycle_nth(Grain::Day, 0)
     );
     b.rule_1_terminal("tomorrow",
-                      b.reg(r#"(?:tmrw?|tomm?or?rows?)"#)?,
+                      b.reg(r#"tomm?or?rows?"#)?,
                       |_| helpers::cycle_nth(Grain::Day, 1)
     );
     b.rule_1_terminal("the day after tomorrow",
-                      b.reg(r#"(?:the )?day after (?:tmrw?|tomm?or?rows?)"#)?,
+                      b.reg(r#"(?:the )?day after tomm?or?rows?"#)?,
```

Comment:
I think that we can skip typos due to sparrowhawk and normalization, we should not get any typos here